src/content/components/BugList/columnTransforms.js

```diff
@@ -99,5 +99,10 @@ export const columnTransforms = {
   },
   priority(priority) {
     return (<span className={priorityStyles[priority.toLowerCase()]}>{priority}</span>);
+  },
+  status(status) {
```

Comment:
Can we hide statuses like reopened/new/assigned since they're not too useful, and add something to indicate github merged?

Reply:
Updated patch to show empty for reopened/new/assigned bugs and added github-merged status